graph/multi/directed_test.go

```diff
@@ -0,0 +1,65 @@
+// Copyright ©2014 The Gonum Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package multi
+
+import (
+	"testing"
+
+	"gonum.org/v1/gonum/graph"
+)
+
+var (
+	directedGraph = (*DirectedGraph)(nil)
+
+	_ graph.Graph    = directedGraph
```

Comment:
Same as above. If the `directedGraph` global variable is not used, then perform assert that the interfaces are implemented using `(*DirectedGraph)(nil)` directly.

Reply:
Ack